engine/engine.go

```diff
@@ -73,6 +77,7 @@ func (e *EngineAPI) ForkchoiceUpdatedV3(
 ) (*eth.ForkchoiceUpdatedResult, error) {
 	e.lock.Lock()
 	defer e.lock.Unlock()
+	defer e.metrics.RecordRPCMethodCall(ForkchoiceUpdatedV3MethodName, time.Now())
```

Comment:
For my understanding:

the `time.Now()` gets called "as this is read", so that start time of the function is available to the metrics recording function.

Reply:
Yep, the start time whenever the RPC method was originally called will be evaluated immediately and passed into `RecordRPCMethodCall` and will then be subtracted from the current time whenever the RPC method completes.